src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/Services/Interfaces/ISupportPackageBuilderService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ISupportPackageBuilderService.cs" company="WildGums">
+//   Copyright (c) 2008 - 2018 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Orc.SupportPackage
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    public interface ISupportPackageBuilderService
```

Comment:
Why is this implemented in the xaml assembly and not in the core one?

Reply:
Because it also require to move some model classes to the core such as SupportPackageDirectory, SupportPackageFileNamePattern, SupportPackageFileSystemArtifact. So, I decided to create a new service at xaml assembly instead move such classes to core.

Reply:
Hmmm, not sure why we put them in the xaml package in the first place, but I agree this is too much of a breaking change.

Reply:
I can put all of to the core package, but just tried to break less as possible

Reply:
Keep as is please, we don't want breaking changes for now.